ipspot/ipv6.py

```diff
@@ -148,6 +148,34 @@ def _ipleak_net_ipv6(geo: bool=False, timeout: Union[float, Tuple[float, float]]
         return {"status": False, "error": str(e)}
 
 
+def _my_ip_io_ipv6(geo: bool=False, timeout: Union[float, Tuple[float, float]]
+                   =5) -> Dict[str, Union[bool, Dict[str, Union[str, float]], str]]:
+    """
+    Get public IP and geolocation using my-ip.io.
+
+    :param geo: geolocation flag
+    :param timeout: timeout value for API
+    """
+    try:
+        data = _get_json_standard(url="https://api6.my-ip.io/v2/ip.json", timeout=timeout)
```

Comment:
Also I'm getting "DNS_PROBE_FINISHED_NXDOMAIN" when opening this link in my browser.